gpii/node_modules/settingsHandlers/src/WebSocketsSettingsHandler.js

```diff
@@ -0,0 +1,154 @@
+/*!
+  GPII Web Sockets Settings Handler
+
+  Copyright 2014 Emergya
+
+  Licensed under the New BSD license. You may not use this file except in
+  compliance with this License.
+
+  You may obtain a copy of the License at
+  https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/*global require*/
+
+(function () {
+
+    "use strict";
+
+    var fluid = require("infusion"),
+        gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.settingsHandlers");
+    fluid.registerNamespace("gpii.settingsHandlers.WebSocketsSettingsHandler");
+
+    // This global object keeps in memory the connected clients.
+    // Clients are added and removed respectively through addBrowser and
+    // removeBrowser functions
+    //
+    gpii.settingsHandlers.WebSocketsSettingsHandler.clients = {};
```

Comment:
Ok - in that case we'll have to have the LifecycleManager locate it from that global path as well. Need to write a big TODO on this since the framework's support for global components is going to change soon and so this will be a fragile point in future when upgrade universal's infusion. Ideally it should be the LifecycleManager which constructs it so that at least it has a well-defined location in the real component tree, and then the LifecycleManager can have an onCreate listener which injects the instance of this component to the "well-known global path" so that the settings handler can find it when it executes.